library/kotlin/src/io/envoyproxy/envoymobile/StreamCallback.kt

```diff
@@ -0,0 +1,55 @@
+package io.envoyproxy.envoymobile
+
+import java.nio.ByteBuffer
+
+interface StreamCallback {
```

Comment:
`StreamCallback`(s) is accurate, but something like `ResponseHandler` might be more intuitive for a random user of the interface? Just a thought. I'm good with it as is though.

Reply:
`ResponseHandler` SGTM, will update Swift